src/type/matrix/DenseMatrix.js

```diff
@@ -524,6 +525,71 @@ export const createDenseMatrixClass = /* #__PURE__ */ factory(name, dependencies
     return this._size.slice(0) // return a clone of _size
   }
 
+  /**
+   * Applies a callback function to a reference to each element of the matrix
+   * @memberof DenseMatrix
+   * @param {Function} callback   The callback function is invoked with three
+   *                              parameters: an array, an integer index to that
+   *                              array, and the Matrix being traversed.
+   *
+   * @return {DenseMatrix} matrix
```

Comment:
Can you remove this `@return` from the comments? Sine `_forEach` doesn't return anything.